mne/gui/_viewer.py

```diff
@@ -291,7 +284,7 @@ def clear(self):  # noqa: D102
     @on_trait_change('scene.activated')
     def plot(self):
         """Add the points to the mayavi pipeline"""
-        _scale = self.scene.camera.parallel_scale if not _testing_mode() else 1
+        _scale = self.scene.camera.parallel_scale
```

Comment:
I thought so, after tentatively removing some `_testing_mode()` checks... but travis looks like I might have been rash in removing all of them...